File: test/fixtures/rust/get_with_form.rs
extern crate reqwest;
use reqwest::blocking::multipart;

fn main() -> Result<(), Box<dyn std::error::Error>> {
    let form = multipart::Form::new()
        .text("from", "test@tester.com")
        .text("to", "devs@tester.net")
        .text("subject", "Hello")
        .text("text", "Testing the converter!");

    let client = reqwest::blocking::Client::builder()
        .redirect(reqwest::redirect::Policy::none())
        .build()
        .unwrap();
    let res = client.post("http://localhost:28139/v3")
        .basic_auth("test", Some(""))
        .multipart(form)
        .send()?
        .text()?;
    println!("{}", res);

    Ok(())
}

File: test/fixtures/ruby/get_with_form.rb
require 'net/http'

uri = URI('http://localhost:28139/v3')
req = Net::HTTP::Post.new(uri)
req.basic_auth 'test', ''

req.set_form(
  [
    [
      'from',
      'test@tester.com'
    ],
    [
      'to',
      'devs@tester.net'
    ],
    [
      'subject',
      'Hello'
    ],
    [
      'text',
      'Testing the converter!'
    ]
  ],
  'multipart/form-data'
)

req_options = {
  use_ssl: uri.scheme == 'https'
}
res = Net::HTTP.start(uri.hostname, uri.port, req_options) do |http|
  http.request(req)
end
